src/app/oauth2/middleware.js

```diff
@@ -9,14 +9,7 @@ const {
 
 module.exports = {
   addAuthParamsToSession: async (req, res, next) => {
-    const authParams = {
-      response_type: req.query.response_type,
-      client_id: req.query.client_id,
-      state: req.query.state,
-      redirect_uri: req.query.redirect_uri,
```

Comment:
FWIW - I have a suspicion this property will be sent over in the clear again, once we start wiring up the error paths.

For now, 🔪